edge/pkg/edged/edged.go

```diff
@@ -313,7 +313,7 @@ func MakeKubeClientBridge(kubeletDeps *kubelet.Dependencies) {
 	client := kubebridge.NewSimpleClientset(metaclient.New())
 
 	kubeletDeps.KubeClient = client
-	kubeletDeps.EventClient = nil
+	kubeletDeps.EventClient = client.CoreV1() // nil
```

Comment:
And there are still some codes need to implement in kubernetes, right? If so, please submit to https://github.com/kubeedge/kubernetes branch `v1.29.6-kubeedge`